File: XXBDataSouce/XXBDataSouce/ViewController.m
//
//  ViewController.m
//  XXBDataSouce
//
//  Created by xiaobing on 16/1/22.
//  Copyright © 2016年 xiaobing. All rights reserved.
//

#import "ViewController.h"
#import "XXBDataSouce.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    NSLog(@"%@",@([[XXBDataSouce new] numberOfDateSouce]));
}

@end

File: XXBMediaPicker_swift/XXBMediaPicker/ViewController.swift
//
//  ViewController.swift
//  XXBMediaPicker
//
//  Created by xiaobing on 16/1/21.
//  Copyright © 2016年 xiaobing. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let rightItem = UIBarButtonItem.init(barButtonSystemItem: UIBarButtonSystemItem.Camera, target: self, action: "p_open")
        
        self.navigationItem.rightBarButtonItem = rightItem
    }
    func p_open(){
        self.navigationController!.presentViewController(XXBMediaPicker(), animated: true) { () -> Void in
            print("opened")
        }
    }
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

